Project-Service-implementation-Script/common_config/config.ini

```diff
@@ -3,7 +3,7 @@
 elevateuserhost = https://project-dev.elevate-apis.shikshalokam.org/
 
 # Host URL for the elevate project service
-elevateprojecthost = https://project-dev.elevate-apis.shikshalokam.org/
+elevateprojecthost = https://project-dev.elevate-apis.shikshalokam.org/project/v1/
```

Comment:
@MallanagoudaB this may break .please check again

Reply:
The code will not break @VISHNUDAS-tunerlabs . host is dev but the enpoint you refering in local.